styles.css

```diff
@@ -0,0 +1,302 @@
+@import url('https://fonts.googleapis.com/css2?family=Crete+Round&family=Domine:wght@700&display=swap');
+@import url('https://fonts.googleapis.com/css2?family=Inter:wght@700&display=swap');
+
+body {
+  margin: 0;
+  padding: 0;
+  width: 100%;
+  height: 125vh;
+  background: #1c1a19;
+  background-size: cover;
+}
+
+.portfolio {
+  position: absolute;
+  width: 100%;
+  height: 125vh;
+  left: 0;
+  top: 0;
+  margin: 0 auto;
+}
+
+.details {
+  display: flex;
+  justify-content: space-around;
+  align-items: center;
+  position: absolute;
```

Comment:
- [ ] I would suggest avoiding the use of `position: absolute` for the main sections of the webpage.